content/projects/metabake.md

```diff
@@ -1,18 +1,19 @@
 ---
 title: Metabake
-repo: metabake/mbake-CLI-for-Metabake
-homepage: http://www.metabake.net
+repo: metabake/MetaBake-Examples
+homepage: http://mBake.org
```

Comment:
This results in some sort of text file being downloaded, no site.

Reply:
I had a typo in DNS. It has to DNS cache expire ~ 30 minutes. In any case I'll keep the site up.